town-hall/2021-10-29.md

```diff
@@ -1,42 +1,116 @@
 
-- Welcomes
-- POAP
-- DAO Stats
-  - Gensis NFT
-    - Total Mints
-    - Holders
-  - Discord
-  - Twitter
+## Introduction
 
-## Action items to review
+- Welcome to our 6th DevDAO Town Hall!
+- Why are we here?
+    - To share updates on open proposals that could impact anything regarding projects, missions, values, goals, and plans for the future of the  DevDAO
+    - It's a good chance for us to come together
+    - To improve our communication as an organization
+- The best place to start if you are new is the [discourse/forum](https://forum.developerdao.com/)
 
-- Community to give final word on
-  [mission, values, goals](https://forum.developerdao.com/t/p-2-defining-our-mission-values-and-goals/137).
-- Community to continue to weigh in on
-  [Season 0 items](https://forum.developerdao.com/t/how-we-move-forward-as-a-dao-season-0/56/27).
-- `willblackburn.eth` to get Snapshot set up for voting.
+## Action Items to review
+
+- Community gave final word on [mission, values and goals snapshot vote](#mission-values-and-goals-snapshot-vote)
+- Community to continue to weigh in on [Season 0 items](#season-0-items).
 
 ## Project Updates
 
+- Genesis NFT: Where are we at?
+        -4,631 minted
+    - 2,748 individual holders
+    - 1,750 in Discord
+    - 1-month old as a DAO
 - Website
+    - https://www.developerdao.com/mint
+        - need some help with the Smart Contract
+        - need help with contract on Rinkeby to test back-end
+        - in search of auditors for Smart Contract - reach out to team on Derivatives channel or to `goldzulu`
 - Derivatives
-- Knowledge base
-- Potential partnerships
+    - `goldzulu` IPFS glitches, worked on front-end of project
 
-## Mission, Values & Goals Snapshot Vote
+## Mission, Values and Goals Snapshot Vote
 
 - Results of our first vote!
+- Community gave final word on [mission, values, goals](https://forum.developerdao.com/t/p-2-defining-our-mission-values-and-goals/137).
+    - 99.5% voted yes on [Snapshot](https://snapshot.org/#/devdao.eth/proposal/QmdZQD8h28PiWwwsdJo5mD2DBdC7BvYzrktmbsMoP4AcP6)
+
+## Season 0 Update
+
+- [Season 0 proposal](https://forum.developerdao.com/t/how-we-move-forward-as-a-dao-season-0/56/27) has received some great feedback as well, giving that more time to integrate into a final proposal. Please weigh in!
+- [Live Season 0 Proposal doc](https://docs.google.com/document/d/1zulImV8eZ_GuOiO-7qHl13vYbHMPiPVO4OF_U2vwG1g/edit) working proposal into a more refined overview of what a season is, guidelines, governance, organizational, etc.
+    - Pushing to a snapshot vote this week
+    - Season 0 = 2 months, with option to extend one month via snapshot vote if required
+        - this is a place for us all to figure it out and set the framework
+    - DAO Structure
+        - Governance Layer
+            - Core Team
+                - Guilds
+                    - Projects
+    - $DBUCKS will be the currency of reward for contributions in the DAO
```

Comment:
```suggestion
    - $DBUCKS will eventually become the native governance token for Developer DAO distributed to members based on their contributions
```

Reply:
Switching up the language to make it a little more accurate regarding positioning of token